src/accel/LocalTransporter.cc

```diff
@@ -190,10 +191,6 @@ void LocalTransporter::Flush()
         return;
     }
 
-    CELER_LOG_LOCAL(info) << "Transporting " << buffer_.size()
-                          << " tracks from event " << event_id_.unchecked_get()
-                          << " with Celeritas";
-
```

Comment:
I removed this entirely, but maybe we should guard it with `celeritas::device`?

Reply:
Yeah. Or at least make it `debug` level, or even something like printing it every N buffered tracks...